app/src/main/java/io/intrepid/skeleton/base/BaseFragment.java

```diff
@@ -0,0 +1,92 @@
+package io.intrepid.skeleton.base;
+
```

Comment:
I personally prefer enforcing all Fragments to use MVP since creating a non mvp BaseFragment might lead to a slippery slope of people forgoing mvp entirely. I don't think there are many instances where we have a view that contains no logic. Even for views that just displays the data, it should go through the presenter if the data comes from the database/server/etc (i.e. not constant)